### ISSUE TITLE
Existing account importer

### ISSUE THREAD

**Issue by kwek20** [iotaledger/iota-java#178]
We need to make an importer to use the account module from an existing seed.
Method signature: ´iotaAccount.importStateFromSeed(boolean overwriteCurrent)´

Boolean is used when we find conflicting balances in the existing account state, and the one from the node.

Does not run by default on anything, and must have a disclaimer on the javadoc that it is unchecked. Developers should not underestimate responsibility for usage of this method.

**Comment by wonkytonky2** [comment]
is this why i'm not getting balance?   there is no import from existing seed?.

IotaAccount account = new IotaAccount.Builder(
					"KIZVHLJALXKEVKLDYARIK.............YOTALRMCNMPABGIQZ")
							.securityLevel(2).mwm(14).depth(2).store(store).api(api).build();

			account.load();

**Comment by kwek20** [comment]
Yes, Accounts are for now intended to use from a clean slate. It is best to generate a new seed, use it on the account to get a new deposit address, and send your balance to this new seed.

Otherwise, you could find the balance for each address and manually add them in the json storage, but that seems like a lot of effort :D
